### ISSUE TITLE
a11y integration tests

### ISSUE THREAD

**Issue by goderbauer** [flutter/flutter#12287]
We should have a way to write integration tests for assistive technologies. These tests should assert that an app is usable with TalkBack (Android) and VoiceOver (iOS) just like they are supposed to.

A framework that enables that is useful for
* ourselves to verify that our a11y implementation interacts correctly with TalkBack and VoiceOver
* all Flutter apps developers to ensure that their support for assistive technologies in their apps doesn't break.

**Comment by eseidelGoogle** [comment]
See also https://github.com/flutter/flutter/issues/11763

**Comment by jonahwilliams** [comment]
We have a (simple) android integration test for accessibility done. Initial investigation of iOS seems infeasible for now.

**Comment by jonahwilliams** [comment]
For the short term, the specific issue for us is that we have no way to turn talkback on or off for a single test, so we need a dedicated device if we don't want to mess up benchmarks
